docs/getting_started/install_manually.md

```diff
@@ -212,290 +212,115 @@ Open your project in the browser and you should see the welcoming page.
 
-``` bash
-cd /usr/lib/php
-curl -O https://pear.php.net/go-pear.phar
-php -d detect_unicode=0 go-pear.phar
-sudo php install-pear-nozlib.phar
-sudo pear channel-update pear.php.net
-sudo pecl channel-update pecl.php.net
-sudo pear upgrade-all
-sudo pear config-set auto_discover 1
-```
+### Download eZ Platform
 
-b. Install autoconf:
+You can download eZ Platform in two ways:
 
-``` bash
-brew install autoconf
-```
+1\. Download an archive
 
-c. Install intl:
+- If you are installing eZ Platform, download the latest archive from [ezplatform.com](https://ezplatform.com/#download-option).
+- For licensed eZ Enterprise customers, download your file from the [Support portal](https://support.ez.no/Downloads).
 
-``` bash
-brew install icu4c
-sudo pecl install intl
-```
+Extract the archive into `/var/www/ezplatform`.
 
-d. The path to the ICU libraries and headers is: `/usr/local/opt/icu4c/`.
+2\. Clone GitHub repository
 
-Edit `/private/etc/php.ini` and add following line:
+You can also clone one of the [repositories from GitHub](#available-distributions).
 
 ``` bash
-extension=intl.so
+git clone https://github.com/ezsystems/ezplatform.git /var/www/ezplatform
 ```
 
-e. Enable opcache extension for PHP (suggested, but not required) by adding:
-
-``` bash
-zend_extension=opcache.so
-```
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
 
-### 9. Install eZ Platform
+!!! tip
 
-a. Go to the folder with your installation and set up directory permissions:
+    You can use any other folder name in place of `ezplatform` in the examples above.
+    You'll point your virtual host to this folder to use as its root.
 
-``` bash
-chmod 775 ../ez1.lh
-chmod 775 ../../workspace
-chmod 775 ../../../Documents
-```
+### Get Composer
 
-b. Download archive from [ezplatform.com](https://ezplatform.com/#download-option). Extract the eZ Platform archive to a directory, then execute post install scripts.
+Install Composer, the PHP command line dependency manager, inside your project root directory by running the following command in the terminal:
 
 ``` bash
-cd /<directory>/
-php -d memory_limit=-1 composer.phar run-script post-install-cmd
+php -r "readfile('https://getcomposer.org/installer');" | php
 ```
 
-c. Copy the virtual host template:
-
-``` bash
-sudo cp doc/apache2/vhost.template /private/etc/apache2/users/ez1.lh.conf
-```
+!!! tip "Install Composer globally"
 
-d. Edit the new virtual host:
+    If you want to install Composer globally, you can use a package manager, for example [Homebrew](https://brew.sh/).
 
-``` bash
-sudo vi /private/etc/apache2/users/ez1.lh.conf
-```
+### Run installation scripts
 
-e. Modify virtual host file **vhost.template.**
+Composer will look inside the `composer.json` file and install all of the packages required to run eZ Platform.
+The `bin/console` script will then install eZ Platform for your desired environment (dev/prod).
```

Comment:
for dev environment or (dev - recommended)? - as we advise to use it with dev only